config/locales/nl.yml

```diff
@@ -1,25 +1,25 @@
 nl:
   edit: Wijzig
-  failure_when_forbidden:
-  feed_latest: RubyGems.org | Recente Gems
+  failure_when_forbidden: Controleer het webadres of probeer het opnieuw.
+  feed_latest: RubyGems.org | Nieuwste Gems
   feed_subscribed: RubyGems.org | Geabonneerde Gems
-  footer_about: RubyGems.org is de gem hosting service van de Ruby community. Publiceer and installeer je gems direct. Gebruik de API om meer informatie van beschikbare gems te vinden. Word een deelnemer en verbeter de site met jouw aanpassingen.
+  footer_about: RubyGems.org is de gem hosting service van de Ruby community. <a href="%{publish_docs}">Publiceer</a> en <a href="%{install_docs}">installeer</a> je gems direct. Gebruik <a href="%{api_docs}">de API</a> om meer informatie over beschikbare gems te vinden. <a href="%{contributing_docs}">Word een deelnemer</a> en verbeter de site met jouw aanpassingen.
   form_disable_with: Een moment geduld...
   help_url: http://help.rubygems.org
   locale_name: Nederlands
-  none:
-  not_found:
+  none: Geen
+  not_found: Niet Gevonden
   please_sign_up: Toegang geweigerd. Schrijf je in voor een account op https://rubygems.org
   sign_in: Inloggen
-  sign_up: Aanmelden
-  multifactor_authentication:
+  sign_up: Registreren
+  multifactor_authentication: Multifactor authenticatie
   subtitle: De gem host voor de community
-  this_rubygem_could_not_be_found: This rubygem could not be found.
+  this_rubygem_could_not_be_found: De gem kon niet gevonden worden.
   time_ago: "%{duration} geleden"
   title: RubyGems.org
   update: Wijzig
-  try_again:
-  advanced_search:
+  try_again: Er is iets fout gegaan. Probeer het opnieuw.
```

Comment:
Maybe this can be: "Er is iets fout gegaan, probeer het opnieuw."

Reply:
Should be fixed now.